docs/getting-started/v3-migration.md

```diff
@@ -69,24 +37,137 @@ Chart.js 3.0 introduces a number of breaking changes. Chart.js 2.0 was released
+
+The v2 configuration below is shown with it's new v3 configuration
+
+```javascript
+options: {
+  scales: {
+    xAxes: [{
+      id: 'x',
+      type: 'time',
+      display: true,
+      scaleLabel: {
+        display: true,
+        labelString: 'Date'
+      },
+      ticks: {
+        major: {
+          enabled: true
+        },
+        fontStyle: function(context) {
+          return context.tick && context.tick.major ? 'bold' : undefined;
+        },
+        fontColor: function(context) {
+          return context.tick && context.tick.major ? '#FF0000' : undefined;
+        }
+      }
+    }],
+    yAxes: [{
+      id: 'y',
+      display: true,
+      scaleLabel: {
+        display: true,
+        labelString: 'value'
+      }
+    }]
+  }
+}
+```
+
+And now, in v3:
+
+```javascript
+options: {
+  scales: {
+    x: {
+      type: 'time',
+      display: true,
+      scaleLabel: {
+        display: true,
+        labelString: 'Date'
+      },
+      ticks: {
+        major: {
+          enabled: true
+        },
+        fontStyle: function(context) {
+          return context.tick && context.tick.major ? 'bold' : undefined;
+        },
+        fontColor: function(context) {
+          return context.tick && context.tick.major ? '#FF0000' : undefined;
+        }
+      }
+    },
+    y: {
+      display: true,
+      scaleLabel: {
+        display: true,
+        labelString: 'value'
+      }
+    }
+  }
+}
+```
+
+#### Animations
 
 Animation system was completely rewritten in Chart.js v3. Each property can now be animated separately. Please see [animations](../configuration/animations.md) docs for details.
 
-* `hover.animationDuration` is now configured in `animation.active.duration`
-* `responsiveAnimationDuration` is now configured in `animation.resize.duration`
+
+#### Customizability
+
+* `custom` attribute of elements was removed. Please use scriptable options
+* The `hover` property of scriptable options `context` object was renamed to `active` to align it with the datalabels plugin.
+
+#### Interactions
+
+* `interactions` are now limited to the chart area
+* `{mode: 'label'}` was replaced with `{mode: 'index'}`
+* `{mode: 'single'}` was replaced with `{mode: 'nearest', intersect: true}`
+* `modes['X-axis']` was replaced with `{mode: 'index', intersect: false}`
+* `options.onClick` is now limited to the chart area
+
+#### Ticks
+
+* `options.ticks.major` and `options.ticks.minor` were replaced with scriptable options for tick fonts.
+* `Chart.Ticks.formatters.linear` and `Chart.Ticks.formatters.logarithmic` were replaced with `Chart.Ticks.formatters.numeric`.
+
+#### Tooltip
+
+* `xLabel` and `yLabel` were removed. Please use `index` and `value`
 
 ## Developer migration
 
 ### Removed
 
+The following properties and methods were removed during v3 development:
```

Comment:
nit: you could probably drop "during v3 development". It's a bit redundant since the whole migration guide is for v3 and you could technically say that in every section

Reply:
Removed